dpp-frontend/src/views/PassportView.vue

```diff
@@ -855,19 +697,36 @@ export default {
             let additionalData = [];
             let sources = [];
             // In order to have the additional data available we need to copy it in deep
-            if (
-              jsonUtil.exists("additionalData", this.data["data"]["aspect"])
-            ) {
-              additionalData = jsonUtil.copy(
-                this.data["data"]["aspect"]["additionalData"]
-              );
+            if (jsonUtil.exists("additionalData", this.data["data"]["aspect"])) {
+              additionalData = jsonUtil.copy(this.data["data"]["aspect"]["additionalData"]);
             }
             // When extend deep is called this property will be replaced
             if (jsonUtil.exists("sources", this.data["data"]["aspect"])) {
               sources = jsonUtil.copy(this.data["data"]["aspect"]["sources"]);
             }
+
+            // process aspect data based on vc or not vc is available
+            // case 1: if verifiable credntials vc is not present in data (vc=false),
+            // then only passport is shown without verification details
+            if (!this.data["verification"]["vc"]) aspect = jsonUtil.get("data.aspect", this.data);
+
+            // case 2: if verifiable credntials vc is present in data (vc=true),
+            // then passport with verification details is shown
+            if (this.data["verification"]["vc"]) aspect = passportUtil.getAspectData(this.data);
```

Comment:
If there is no "verification" in the data there is an exception no?

Reply:
fixed